src/stores/quiz.ts

```diff
@@ -8,15 +8,20 @@ export interface QuizItem {
 	answered?: string
 	incorrectTimes?: number
 	isCorrectAnswer?: boolean
+	duration?: number
 }
 
 export interface Quiz {
+	time: number
```

Comment:
Why store the current time here? Is this maybe more like `startTime`? In any case, this could be `derived` instead of being calculated manually

Reply:
This time was meant to keep an accurate total session time (since the user could visit the session summary screen and then resume the quiz). It would add the time difference to quiz `duration` when they visit the summary screen, and then reset the `time` to the current time in order to track the next portion that they were in the quiz screen.

A much cleaner solution that I changed this to was to determine the session time by just summing up the kana durations.